### PR TITLE
Add some discussion about future directions

### PR DESCRIPTION
**Pull request by eteq**
From some referee feedback, this PR adds a paragraph about the "future".  Basically a description of how future directions are *intentionally* led by contributors rather than roadmapped/architected.

(Also adds STScI as an acronym, as I noticed it's missing)

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -415,12 +416,19 @@ \section{Development history and status}
...
 project\footnote{\url{http://sbpy.org}} with support from NASA PDART grant
 80NSSC18K0987. Further Solar System-related services are planned to be added to
-\astroquery through this support.
-
-\astroquery has received support from the Google Summer of Code
+\astroquery through this support. \astroquery has also received support from the Google Summer of Code
 program, with two students (co-authors Madhura Parikh and Simon Liedtke)
 from 2013--2014.
 
+Due to its nature as a open developed ``big tent'' package, \astroquery intentionally
```
Comment (keflavich) [inline main.tex]: I don't like "big tent"; I had never heard the colloquialism and had to look it up
```suggestion
Due to its nature as an openly developed package, \astroquery intentionally
```

File: main.tex [hunk 2]
```diff
@@ -415,12 +416,19 @@ \section{Development history and status}
...
+does not have a long-term roadmap for future features. New directions are primarily 
+driven by contributors and data providers adding or updating modules to reflect new 
+or changed data  sources. The underlying software architecture has been demonstrably
+sufficient to meet the needs of the  current generation of data sources (proven by  
+the user base of \astroquery).
+While this may change in the future, the user-focused nature of \astroquery means
+that making such changes is unnecessary until there are specific data sources
```
Comment (keflavich) [inline main.tex]:
```suggestion
that making such changes is unnecessary until there are specific data sources or use cases
```

File: main.tex (hunk 1 above)
Comment (bsipocz) [reply main.tex]: agreed

File: main.tex (hunk 2 above)
Comment (eteq) [reply main.tex]: WHOA! This "suggested change" thing is blowing my mind!

**Comment by eteq** [comment]
Ok that "suggested change" beta feature is pretty neat.  The one downside I guess is that it makes a commit per change which is a bit of a CI nightmare... but still!

(Anyway I think this should now be good)

**Comment by bsipocz** [comment]
@eteq - it's called a beta for a reason ;)

**Pull request merged**